src/components/attendee/Button/Button.module.scss

```diff
@@ -28,3 +33,9 @@
   background: transparent;
   border: 1px solid white;
 }
+
+.buttonPrimaryAlternate {
+  @include sparkle.color--glass;
+  background: transparent;
+  backdrop-filter: none;
+}
```

Comment:
There is another way to make combinations of classes:

`.primary.alternate`, `.alternate.border` ...

Reply:
redone [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)